src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -730,6 +730,11 @@ public function queue_import( $args = array() ) {
 				: date( Tribe__Date_Utils::DBDATETIMEFORMAT, $args['end'] );
 		}
 
+		// Set site for origin(s) that need it for new token handling.
+		if ( 'eventbrite' === $args['origin'] ) {
+			$args['site'] = site_url();
```

Comment:
Could we fetch that from the Callback URL? or Other data that we might have previously?

Reply:
Previews don’t send callback. This just standardizes a param to send either way. I don’t want to send it for non EB token requests because it’d start adding lots of sites to the DB unnecessarily.